faiss/gpu/GpuDistance.cu

```diff
@@ -237,13 +232,33 @@ void bfKnn(GpuResourcesProvider* prov, const GpuDistanceParams& args) {
             "limitation: both vectorType and queryType must currently "
             "be the same (F32 or F16");
 
-#if defined USE_NVIDIA_RAFT
-    // Note: For now, RAFT bfknn requires queries and vectors to be same layout
-    if (should_use_raft(args) && args.queriesRowMajor == args.vectorsRowMajor) {
-        DistanceType distance = metricFaissToRaft(args.metric, false);
+#if defined USE_NVIDIA_CUVS
+    // Note: For now, cuVS bfknn requires queries and vectors to be same layout
```

Comment:
I believe we can have mixed layout with cuvs now - this comment might be out of date